resources/resources.go

```diff
@@ -0,0 +1,51 @@
+package resources
```

Comment:
nit: Could we rename this package to `core`? I could imagine other things possibly belonging here that could be generally shared, but relate more to the main "hauser" process.

Ideally, I think these would actually be in a root "hauser" package and the main package would actually be located in `cmd/hauser`. Alas...

Reply:
Done.